documentation/modules/tracing/proc-instrumenting-kafka-streams-with-tracers.adoc

```diff
@@ -1,27 +1,56 @@
 // Module included in the following assemblies:
 //
-// assembly-instrumenting-kafka-clients-tracers.adoc
+// assembly-distributed tracing.adoc
 
 [id='proc-instrumenting-kafka-streams-with-tracers-{context}']
 = Instrumenting Kafka Streams applications for tracing
 
-This section describes how to instrument Kafka Streams API applications for distributed tracing.
+[role="_abstract"]
+Instrument Kafka Streams API applications for distributed tracing.
+OpenTelemetry and OpenTracing instrumentation projects provide classes that support instrumentation of Kafka Streams.
```

Comment:
If we are referring to the `TracingKafkaClientSupplier` class, this is truer for OpenTracing but not for OpenTelemetry where the class is not provided out of box but we explain how to create it in the producer.

Reply:
Changed to:

> The OpenTracing instrumentation project provides a `TracingKafkaClientSupplier` class that supports instrumentation of Kafka Streams.
> For OpenTelemetry, you need to create a custom `TracingKafkaClientSupplier` class to provide the support.